src/scrape_up/github/pull_request.py

```diff
@@ -69,3 +69,20 @@ def files_changed(self):
         except:
             Message = "No files changed found"
             return Message
+
+    def reviewers(self):
+        """
```

Comment:
Include this method in `documentation.md` as well

Reply:
I just need to update the documentation ryt?

Reply:
Yes.